kakao/src/main/kotlin/com/agoda/kakao/edit/TextInputLayoutAssertions.kt

```diff
@@ -34,6 +37,15 @@ interface TextInputLayoutAssertions : BaseAssertions {
         })
     }
 
+    fun hasHint(@StringRes resId: Int) {
+        hasHint(getResourceString(resId))
+    }
+
+    private fun getResourceString(@StringRes resId: Int): String {
+        val targetContext: Context = InstrumentationRegistry.getInstrumentation().targetContext
+        return targetContext.resources.getString(resId)
+    }
```

Comment:
Just a nerdy question. WDYT about

private fun getResourceString(@StringRes resId: Int) = InstrumentationRegistry.getInstrumentation().targetContext.let { it.resources.getString(resId) }

@Unlimity can we create some object class for internal usage? F.e in `common/utility/ContextUtils` and move this function into with `internal` modifier?

Reply:
Regarding "full/long version" of `fun hasError(@StringRes resId: Int) {` - I don't really see any benefit of doing so (instead of using one overload with in another).  

However on the other side we have a big downside of heaving two full/long implementations - higher maintenance cost (using overload).